sdk/forms/src/main/java/com/klaviyo/forms/webview/KlaviyoWebViewClient.kt

```diff
@@ -54,6 +55,9 @@ internal class KlaviyoWebViewClient(
         Registry.log.debug("Klaviyo webview is already initialized")
     } ?: KlaviyoWebView().let { webView ->
         val nativeBridge: BridgeMessageHandler = Registry.get()
+        val observerCollection = Registry.get<ObserverCollection>()
+        val handshake = nativeBridge.handshake + observerCollection.handshake()
```

Comment:
Handshake is now a compiled list of 
- Native bridge messages, from JS -> SDK
- Injected data from SDK -> JS.